bundles/net.tourbook/src/net/tourbook/ui/views/tourMarker/TourMarkerView.java

```diff
@@ -658,6 +660,36 @@ public void update(final ViewerCell cell) {
 
    }
 
+   private void defineColumn_Body_AvgPulse() {
+      final ColumnDefinition colDef = TableColumnFactory.BODY_AVG_PULSE.createColumn(_columnManager, _pc);
+
+      colDef.setLabelProvider(new CellLabelProvider() {
+         @Override
+         public void update(final ViewerCell cell) {
+
+            final ViewerRow lastRow = cell.getViewerRow().getNeighbor(ViewerRow.ABOVE, false);
```

Comment:
this is an interesting code, I've never used `getNeighbor()`

Reply:
I thought I had gotten it from [your code](https://github.com/wolfgang-ch/mytourbook/blob/9ce4b06b1ac39e6a1341b248ab996ec4ef312c66/bundles/net.tourbook/src/net/tourbook/ui/views/tourMarker/TourMarkerView.java#L796) actually :-)

Reply:
too much code in my brain -> details get lost :-(